core/src/main/java/io/questdb/griffin/engine/functions/bool/InStrFunctionFactory.java

```diff
@@ -69,7 +69,7 @@ public Function newInstance(
                 case ColumnType.STRING:
                 case ColumnType.VARCHAR:
                 case ColumnType.SYMBOL:
-                    if (func.isRuntimeConstant()) { // bind variables
+                    if (func.isRuntimeConstant() && !func.isNullConstant()) { // bind variables
```

Comment:
I guess we need a similar fix in `InVarcharFunctionFactory` and a test around that change.